apollo/uma/queries.ts

```diff
@@ -11,6 +11,27 @@ export const ACTIVE_POSITIONS = gql`
           id
         }
       }
+      liquidations {
```

Comment:
so query all this information, every time. do we have any understanding of how much data this query is now transferring? not really an issue with 200 or 300 sponsors but might become a bit of an issue if we scale up more and need to improve the pagination of the table to query more data as you move between pages.

Reply:
I think it will do the same as the `positions` query which is to query the first 100 by default. I didn't worry about this for now because I don't expect there to be many liquidations relative to sponsor positions. WDYT?

I will explicitly set this to `first: 1000` for now just in case